lib/poker_hands/hand_rankings/pair.rb

```diff
@@ -1,5 +1,10 @@
 class Pair
   def check(hand)
-    true
+
```

Comment:
⚒ Generally we avoid whitespace as the first line of a class or method (we have a linter setup that would complain about this in most of our projects).